Code/CheXpertDataset.py

```diff
@@ -66,14 +66,12 @@ def __call__(self, sample):
 class UZerosTransform:
     def __call__(self, sample):
         sample = sample[label_columns].replace(-1, 0)
-        sample = sample[label_columns].fillna(0)
         return sample

```

Comment:
realized these lines aren't needed. see label_processing_uzeros and uones in the common_training file

Reply:
Lol I meant to delete them from the other file when I changed this.  But since we're always changing blanks to 0s, this works too!

Reply:
Oh wait, you changed it in the other file.  Glad it wasn't an oversight on my part

Reply:
I only changed it because for some reason it wasn't working before. don't remember why. but the NaN remover worked... so i simply composed them together... it also seems a bit cleaner.